helm/falco-app/values.yaml

```diff
@@ -23,9 +23,37 @@ falco:
       enabled: true
     grpc_output:
       enabled: true
+    rules_file:
+    # adding the rule file list to override the default, because the default does not include plugin rules.
+      - /etc/falco/falco_rules.yaml
+      - /etc/falco/falco_rules.local.yaml
+      - /etc/falco/rules.d
+      - /etc/falco/k8s_audit_rules.yaml
```

Comment:
I don't fully understand this one, which are the implications of overriding the default rules?

Reply:
I've added this list because by default it only includes the first three lines.
But if I only put down the k8s_audit_rules.yaml file there, it would override the rules_file field and tell the application to only use that rule-set? Right?